content/en/docs/releases/bugs/index.md

```diff
@@ -65,16 +69,40 @@ containing:
 
 * Current and previous logs from all Istio components and sidecar
 
-* Istiod logs:
+    * Istiod logs:
+
+        {{< text bash >}}
+        $ kubectl logs -n istio-system -l app=istiod
+        {{< /text >}}
+    
+    * Ingress Gateway logs:
+    
+        {{< text bash >}}
+        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l istio=ingressgateway -n $ns ; done
+        {{< /text >}}
+    
+    * Egress Gateway logs:
+    
+        {{< text bash >}}
+        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l istio=eressgateway -n $ns ; done
```

Comment:
```suggestion
        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l istio=egressgateway -n $ns ; done
```

Although in general "ingressgateway" and "egressgateway" are just some common names - we will likely see these be less common